command/job_start.go

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/posener/complete"
+)
+
+type JobStartCommand struct {
+	Meta
+	versionSelected uint64
+}
+
+func (c *JobStartCommand) Help() string {
+	helpText := `
+Usage: nomad job start [options] <job>
+Alias: nomad start
+
+  Start an existing stopped job. This command is used to start a previously stopped job's
+  most recent version. Upon successful start, an interactive
+  monitor session will start to display log lines as the job starts its
+  allocations based on its most recent version. It is safe to exit the monitor
+  early using ctrl+c.
```

Comment:
Nitpick: the output of these commands are what get printed literally to the terminal. There's no automatic reflowing in `mitchellh/cli`. So we should try to manually reflow these to 80 cols.